overview/content-types/project-list-item.nix

```
@@ -17,26 +17,7 @@ in
       type = with types; nullOr str;
     };
     deliverables = mkOption {
-      type =
-        with types;
-        listOf (submodule {
-          options = {
-            name = mkOption {
-              type = str;
-            };
-            type = mkOption {
-              type = enum [
-                "program"
-                "service"
-                "demo"
-              ];
-            };
-            hasProblem = mkOption {
-              type = nullOr bool;
-              default = null;
-            };
-          };
-        });
+      type = with types; listOf (submodule ./deliverable.nix);
```

```suggestion
      type = with types; listOf (submodule ./project-list-item-tag.nix);
```

The naming of the modules should be catered towards presentation, not our deliverable data model. Imagine a maintainer that looks at a certain UI element (the tags in the items of the project list) and tries to find the correct content-type module for that.